_stbt/transition.py

```diff
@@ -30,7 +37,8 @@ def press_and_wait(
     region: Region = Region.ALL,
     timeout_secs: float = 10,
     stable_secs: float = 1,
-    min_size: SizeT = None,
+    min_size: "SizeT|None" = None,
+    frames: "typing.Iterator[core.Frame]|None" = None,
```

Comment:
I suppose you had to add `|None` to satisfy vscode / pylance?

Interesting that Sphinx automatically works out that it's Optional, even if you don't specify it. See https://stb-tester.com/manual/python-api#stbt.detect_motion vs. https://github.com/stb-tester/stb-tester/blob/1894d831b93ab28a77f527947d2b041d745e6fbd/_stbt/motion.py#L15-L21